### PR TITLE
XRDSD-249: Add information about updating EHAK classifiers and the update file

### PR DESCRIPTION
**Pull request by raits**

### CODE REVIEW

File: docs/misp2_installation_manual_18.04.md [hunk 1]
```diff
@@ -729,6 +731,19 @@ This is due to browsers currently not supporting `post-handshake authentication`
...
 * https://bugzilla.mozilla.org/show_bug.cgi?id=1511989
 
+### 5.8 Updating EHAK classifiers to EHAK2021v4
+
+**NB!** The current EHAK classifiers in MISP2 are specific to Estonia, if you are using MISP2 in another country or are
+not using the classifiers, you do not need to run the update.
+
+To update the EHAK classifiers in the database, please follow these steps:
+
+* Download the update file [EHAK2021v4.sql](./EHAK2021v4.sql) and move it to an appropriate location on the MISP2
```
Comment (petkivim) [inline docs/misp2_installation_manual_18.04.md:741]: There's a dot (`.`) missing at the end of the sentence:

> Download the update file [EHAK2021v4.sql](./EHAK2021v4.sql) and move it to an appropriate location on the MISP2 server.

File: docs/misp2_installation_manual_18.04.md [hunk 2]
```diff
@@ -729,6 +731,19 @@ This is due to browsers currently not supporting `post-handshake authentication`
...
+  server
+* Execute the following command on the server: `psql -p 5432 misp2db -U postgres -f /path/to/EHAK2021v4.sql`
+  * If you chose a different database name during installation, substitue `misp2db` with that. If you are unsure, the
+    details can be checked from `/var/lib/tomcat8/webapps/misp2/WEB-INF/classes/config.cfg`
```
Comment (petkivim) [inline docs/misp2_installation_manual_18.04.md]: There's a dot (`.`) missing at the end of the sentence:

> If you chose a different database name during installation, substitue `misp2db` with that. If you are unsure, the details can be checked from `/var/lib/tomcat8/webapps/misp2/WEB-INF/classes/config.cfg`.

**Review by petkivim — CHANGES_REQUESTED** [review]
Just a couple of very minor fixes. Otherwise looks good to me.

File: docs/misp2_installation_manual_18.04.md (hunk 1 above)
Comment (raits) [reply docs/misp2_installation_manual_18.04.md:741]: Thanks, fixed.

File: docs/misp2_installation_manual_18.04.md (hunk 2 above)
Comment (raits) [reply docs/misp2_installation_manual_18.04.md]: Thanks, fixed.

**Review by petkivim — APPROVED** [review]
Looks good to me.

**Pull request merged**